src/js/utils/dom.js

```diff
@@ -215,5 +215,11 @@ export function openLink(link, target, additionalOptions = {}) {
     a.href = link;
     a.target = target;
     a = Object.assign(a, additionalOptions);
-    a.dispatchEvent(new MouseEvent(`click`, { bubbles: true, cancelable: true, view: window }));
+
+    // Must support all browsers.
+    try {
```

Comment:
is there any way of detecting custom event support rather than a try catch?

Reply:
yes, but not in a way that works for both IE and firefox. I could use document.customEvent as a check, but the property exists on both IE and firefox. a.click() won't work on firefox because the anchor is never appended to the DOM

Reply:
Implemented an OS check instead of try/catch